### PR TITLE
Add Merging Fee FAQ

### PR DESCRIPTION
**Pull request by catenocrypt**

### CODE REVIEW

File: assets/merging-fee.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
+# Merge Fee -- FAQ
+
+**Someone is asking for cypto payment for merging my Pull Request.  Is this a scam?**
```
Comment (vikmeup) [inline assets/merging-fee.md:3]: don't think this part is needed

File: assets/merging-fee.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+**Why?**
+
+Trust Wallet supports an insane number of tokens and coins.
+We are committed to support tokens used by our users.
+But supporting so many assets has a significant costs, in terms of running the infrastructure, ensuring continuous integrations,
+handling a huge number of requests to the assets info, etc.  A lot of spam/scam coins also find us.
+
+We have introduced this crypto contribution in order to filter down requests, so we have more time to handle genuine ones.
+
+**How  dare you ask for money in the case of an Open Source project?**
```
Comment (vikmeup) [inline assets/merging-fee.md:19]: Can we remove this section and use Curated Token Registry format: https://medium.com/@tokencuratedregistry/a-simple-overview-of-token-curated-registries-84e2b7b19a06 ?

File: assets/merging-fee.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+When the `merge-fee-bot` detects payment, it automatically places an Accept review on the PR.
+This is a precondition for merge.
+But merging is not automatic, it is done by the maintainers.
+
+**I just need to make an urgent fix / just change existing info / etc. -- Can you do it without a payment?**
+
+In justified cases, we may waive the payment requirement.
+Make sure to communicate the justification clearly in the PR.
+
+**I have forgot to set a memo in my transfer**
```
Comment (vikmeup) [inline assets/merging-fee.md:53]: Let's avoid manual work. No memo - no review.

File: assets/merging-fee.md [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+
+We warned you not to forget!
+But if you did, don't worry -- write down this fact in a comment in the PR, together with the TX hash, and we will check manually.
+
+**If you decide not to merge my PR, do I get my payment back?**
+
+It depends on the situation.
+If the PR has to be repeated for some techical reason, the payment can be reused for the second PR.
```
Comment (vikmeup) [inline assets/merging-fee.md:61]: let's keep it simple 1 payment per 1 review. let's avoid any manual work - this needs to be automated as much as possible

File: assets/merging-fee.md [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+We are committed to support tokens used by our users.
+But supporting so many assets has a significant costs, in terms of running the infrastructure, ensuring continuous integrations,
+handling a huge number of requests to the assets info, etc.  A lot of spam/scam coins also find us.
+
+We have introduced this crypto contribution in order to filter down requests, so we have more time to handle genuine ones.
+
+**How  dare you ask for money in the case of an Open Source project?**
+
+We believe in the power of open source software, and important parts of Trust Wallet are open source to contribute to the community
+(wallet-core, blockatlas).
```
Comment (vikmeup) [inline assets/merging-fee.md:22]: main goal to increase quality and moderation of supplied data

File: assets/merging-fee.md (hunk 1 above)
Comment (catenocrypt) [reply assets/merging-fee.md:3]: Well, this is the question received most often so far, I think it makes our life easier.

**Comment by catenocrypt** [comment]
I have made some changes according to the comments.
i have also called it 'pull request fee' instead of 'merging fee', as in case of rejected PRs there is no merge.

**Pull request merged**